backend.md

```diff
@@ -1,78 +1,76 @@
-We have some requirements that you must pay attention:
-
-1. The `address` is defined by the format `GeoJSON Point` (https://en.wikipedia.org/wiki/GeoJSON)
-2. The `coverageArea` is defined by the format `GeoJSON MultiPolygon` (https://en.wikipedia.org/wiki/GeoJSON) 
-3. The `document` must be a unique field (as it is the CNPJ)
-
-## What we want you to do
-
-Considering all the info we gave to you, we expect you to develop a service that provides API using REST or GraphQL and enable the following functionalities. The programming language and the database you choose to use it is entirely up to you. 
-
-### 1. Create partner: 
+### 1.2. Load partner by id:
+Return a specific partner by its `id` with all the fields presented above.
 
-All fields are required and must follow the rules set above.
+### 1.3. Search partner:
+Given a specific location (coordinates `lng` and `lat`), search the **nearest** partner **which the coverage area includes** the location.
 
-### 2. Get partner by id:
-
-Get a specific partner by its `id`.
-
-### 3. Search partner:
-
-Given a specific location (coordinates `lng` and `lat`), search the nearest partner considering each partner's coverage area.
-
-### 4. Tech Requirements, Docs and Deploy:
-
-* Your project must be **cross-platform**.
-* Provide a documentation so we know how to execute your service locally and how to deploy it (*focus on simplicity, and don't forget that we should test your service on our own, without further assistance*)
+### 1.4. Technical Requirements:
+* The programming language and the database engine it is entirely up to you;
+* Your project must be **cross-platform**;
+* Provide a documentation (README.md) file explaining how how to execute your service **locally** and how to deploy it (*focus on simplicity, and don't forget that we should test your service on our own, without further assistance*).
 
 ## Evaluation Method
 
-Your code will be under review of the Zx's Global Engineering team. What we will look for:
-- **Performance**
-- **Testing**
-- **Maintanability**
-- **Separation of concerns**
-- **Software engineering**
+We will evaluate your code challenge based on some *ilities.
+Some we consider a must have, like **correctness** and will be evaluated on a boolean (works/follows or not) approach.
+The others, since they are not that objective, will be evaluated by more then one developer and on a scale from 1 to 10.
+This are all the *ilities that we expect you adress:
+- **Correctness:** Your code must follow **all** the requirements presented on item [1.](#1-what-we-want-you-to-do);
+- **Performance:** The more partners you can handle and the fastes you can query the best;
+- **Testability:** How tested and how easy is to add tests to your code;
+- **Maintanability:** How easy to add extra features to your code;
+- **Separation of concerns:** (https://en.wikipedia.org/wiki/Separation_of_concerns)
+- **Software engineering**<!-- TODO: O que queremos dizer com isso?-->
```

Comment:
the portuguese wikipedia page has a better definition than the english version. [Engenharia de software](https://pt.wikipedia.org/wiki/Engenharia_de_software)

Reply:
that article is pretty cool: [What is Software Engineering? Definition, Basics, Characteristics](https://www.guru99.com/what-is-software-engineering.html)